EIPS/eip-1.md

```diff
@@ -42,19 +42,23 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
+Standards track EIPs of type 'Core' are updated from 'Draft' to 'Accepted' status when a core devs meeting has accepted the draft and the participants have announced their intention to implement the EIP for inclusion in a future hard fork. At that point, an EIP number is assigned by the editors and the EIP is moved to the EIPS directory and renamed accordingly. Once a Core EIP has been implemented in at least three clients, and all pass a common set of test suites, the status of the EIP is updated to 'Final'. An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP.
 
-Standards Track EIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification]. The EIP should be reviewed and accepted before an implementation is begun, unless an implementation will aid people in studying the EIP. Standards Track EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final.
+Standards track, non-core EIPs remain in Draft status until at least one implementation has been completed and referenced in the Implementations section of the draft. At that point, the draft's author may apply to the EIP editor, by way of a pull request, to assign a number to the EIP and update its status to 'Final', moving it to the EIPs directory and renaming it accordingly.
 
-For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+Non-standards track EIPs may be transitioned to 'Final' status when the author requests it, and at the discretion of the editors.
 
-Once an EIP has been accepted, the implementations must be completed. When the implementation is complete and accepted by the community, the status will be changed to “Final”.
+For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly. The EIP editor will not unreasonably deny an EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation, or failing to address backwards compatibility.
 
```

Comment:
Some of the stuff in this paragraph feels pretty subjective like "complicate the protocol unduly" and "net improvement".  Casper is a great example of something that is incredibly "complicated", but probably will make it through the EIP process (if it actually used the EIP process) while the whole locked/lost fund recovery stuff is disputed as to whether or not it is a "net improvement".  I think the "reasons to not merge" is pretty good, with the only piece at risk of being contentious due to subjectivity being "proper motivation".